Add 'Results group name' parameter to Configuration section (#6785)
qgis/QGIS-Documentation: docs/user_manual/processing/configuration.rst

  Notice that, if the output is saved to a temporary file, the filename
  of this temporary file is usually a long and meaningless one intended
  to avoid collision with other already existing filenames.
* :guilabel:`Results group name`.
  If you want to obtain all processing result layers in a group in the
  :guilabel:`Layers` panel, set a group name for this parameter. The group
  may exist already or not. QGIS will add all output layers to such a group.

>   By default, this parameter is empty, so all output layers are added to 
>   different places in the :guilabel:`Layers` panel, depending on the layer
>   that is active when running an algorithm.

Reviewer: ```suggestion
  By default, this parameter is empty, so all output layers are added 
  above the layer that is active in the :guilabel:`Layers` panel
  when running an algorithm.
```
Or are there other scenarios of placement?

Author: You question made me think about it. From time to time, the layers panel has some surprises:

If you have no active layer (for that, right click on an empty region of the layers panel) the output layer is added to the top of the layers panel.

Additionally, if the active node is a group, the output layer will be loaded inside that group.

That's why I didn't want to say anything that were too specific :)

Reviewer: Got it. Actually, I think it follows the normal rules of layer loading (at the topmost, at the top or in the group). Not specific to Processing output.

Author: That's right. Just let me know if you still want us to commit your suggestion.
I'd be fine with any decision here.

Reviewer: No. As is, my suggestion was wrong. I updated it to replace layer.
We could mention how loading layers behave (in layers panel or in opening data section) and refer to it. But not necessarily in this pr.

Author: Je suis d'accord.
